src/Components/WebView/test/E2ETest/WebViewManagerE2ETests.cs

```diff
@@ -39,7 +40,9 @@ public async Task CanLaunchPhotinoWebViewAndClickButton()
 
         var testProgramOutput = photinoProcess.StandardOutput.ReadToEnd();
 
-        await photinoProcess.WaitForExitAsync().TimeoutAfter(TimeSpan.FromSeconds(30));
+        await photinoProcess.WaitForExitAsync().TimeoutAfter(TimeSpan.FromSeconds(60));
+
+        output.WriteLine(testProgramOutput);
```

Comment:
Always log the test output even on successful runs to help compare successful runs to failed runs (in case it turns out to be a machine slowness issue).